### PR TITLE
add az aws resource doc

### PR DESCRIPTION
**Pull request by idan-aviv**

### CODE REVIEW

File: cloudrail/knowledge/context/aws/ec2/availability_zone.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 class AvailabilityZone(AwsResource):
 
+    """
```
Comment (yi2020) [inline cloudrail/knowledge/context/aws/ec2/availability_zone.py:9]: Also need to add this file to the README's in the context documentation

**Review by yi2020 — CHANGES_REQUESTED** [review]

File: cloudrail/knowledge/context/aws/ec2/availability_zone.py (hunk 1 above)
Comment (idan-aviv) [reply cloudrail/knowledge/context/aws/ec2/availability_zone.py:9]: are you talking about i.e. docs/context/aws/<aws-resource>/README.md?
if so i see it is related to TF only, az aws resource fetched only from real environment

Comment (yi2020) [reply cloudrail/knowledge/context/aws/ec2/availability_zone.py:9]: The documentation is for the entire context model - the classes and their fields - it doesn't matter if they are built from TF, live or CFN.

File: docs/context/aws/availability_zone/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## ::: cloudrail.knowledge.context.aws.ec2.availability_zone.py
```
Comment (yi2020) [inline docs/context/aws/availability_zone/README.md:1]: Wrong path. Should be aws/ec2

**Pull request closed without merging**